src/test/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -180,7 +180,24 @@ sourceSets {
         return GradleRunner.create()
                 .withProjectDir(testProjectDir.getRoot())
                 .withArguments(arguments)
+                .withGradleVersion(getGradleVersion())
                 .withPluginClasspath()
                 .build()
     }
+
+    abstract String getGradleVersion();
+}
+
+class ScalafixPluginFunctionalTestGradle6 extends ScalafixPluginFunctionalTest {
```

Comment:
I'd prefer the `GradleX` suffix to be prefix instead. Maybe we could change that later